hub/demo/src/components/AgentWelcome.tsx

```diff
@@ -15,10 +17,20 @@ export const AgentWelcome = ({ details }: Props) => {
   if (!welcome?.title || !welcome?.description) return null;
 
   return (
-    <Container size="s" style={{ margin: 'auto', textAlign: 'center' }}>
-      <Flex direction="column" gap="m" align="center">
-        <Text size="text-l">{welcome.title}</Text>
-        <Text>{welcome.description}</Text>
+    <Container size="s" style={{ margin: 'auto' }}>
+      <Flex
+        direction="column"
+        gap="m"
+        style={{
+          borderImageSource:
+            'linear-gradient(to bottom, var(--green-9), var(--violet-9))',
+          borderImageSlice: 1,
+          borderLeft: '2px solid',
+          paddingLeft: 'var(--gap-l)',
+        }}
+      >
+        {welcome.title && <Text size="text-l">{welcome.title}</Text>}
+        <Markdown content={`How can I help you today?`} />
```

Comment:
I think you lost `description` here. Was it intentional?

P.S. I am not stalking, I was just searching for the `welcome` section documentation and had to go to the source code since there is no documentation